### ISSUE TITLE
Provide a good user experience for users with overscan

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/firefox-tv#1055]
### Why/User Benefit/User Problem
Some users have overscan enabled on their TV so they're unable to see the full UI. Fire TV guidelines suggest taking this into account for your app and offsetting your UI so it isn't in the zone where users with overscan can't see it. However, we have no control over web content and web content can fall into this dead zone, provided a bad experience.

We should figure out a way to make sure users with overscan have a good experience. Some options:
- Provide a configuration that allows users to change the content display area for web content (I think Silk does this)
- Educate users about overscan and potentially how to turn it off (because I don't think it's useful? but there's more friction here since each TV is different)
- Provide good defaults

### What / Requirements
- Provide a better user experience for users with webcontent

**Comment by bbinto** [comment]
@mcomella - how can we measure this? We want to first figure out if it's worth doing?

**Comment by mcomella** [comment]
I don't think this is possible to measure in the app without doing something ridiculous. However, there may be available statistics on how many users run their TVs with overscan.

**Comment by mcomella** [comment]
For an example of overscan, see:

![overscan example](https://user-images.githubusercontent.com/102331/46155388-2b501280-c245-11e8-840a-800bcf6febb9.jpeg)

**Comment by mcomella** [comment]
On the Fire TV system, there does not appear to be any way to override overscan: when opening Fire TV Settings -> Display -> Calibrate Display, you will see:

![device-2019-08-07-131456](https://user-images.githubusercontent.com/759372/62654822-88146300-b915-11e9-8c39-97b791fc7882.png)

But it provides no ability to change the position of the screen.

I also remember seeing an option to do this in Silk but I can no longer find it – perhaps it's been removed.

FYI @athomasmoz – it seems the Fire TV & the Silk team is assuming TV owners will correct overscan through their TV settings.
